full-service/src/json_rpc/balance.rs

```diff
@@ -70,3 +70,47 @@ impl From<&service::balance::Balance> for Balance {
         }
     }
 }
+
+/// The "balance" for a view-only-account, as well as some information about
+/// syncing status needed to interpret the balance correctly. In order for the
+/// balance to be accurate, you must mark view_only_txos as spent
+#[derive(Deserialize, Serialize, Default, Debug, Clone)]
+pub struct ViewOnlyBalance {
```

Comment:
So, this will eventually have a field to represent the calculated balance, yeah? Maybe that can wait until we get into implementing tracking spent txos.

Reply:
yeah, that's my plan